playbooks/common/openshift-cluster/upgrades/v3_9/upgrade_control_plane.yml

```diff
@@ -135,6 +135,8 @@
       enabled: true
     when: openshift.common.rolling_restart_mode == 'system'
 
+- import_playbook: label_nodes.yml
```

Comment:
This code looks idempotent, and doesn't actually target any nodes in inventory, just applies labels.

What about running this at the top of post_control_plane.yml?